src/components/ConfirmDialog/ConfirmDialog.vue

```diff
@@ -10,8 +10,8 @@
 
     <v-card>
       <v-card-title>{{ title }}</v-card-title>
-      <v-card-text class="confirm-title pre-wrap" v-show="!!message" v-html="message" />
-      <v-divider class="confirm-text my-0" v-show="!!options.yes || !!options.no || !!options.cancel" />
```

Comment:
Other than the unit tests, hopefully noone else used these classnames since they're attached to the wrong elements.
Anyway, a v-card attaches its own classnames to these elements -- see unit test for example.